compiler/passes/insertWideReferences.cpp

```diff
@@ -1989,5 +2079,8 @@ insertWideReferences(void) {
   }
   printf("Summary:\n\twide: %d\n\tnarrow: %d\n", wide, narrow);
 #endif
+
+  printCauses(NULL);
```

Comment:
I think an ifdef or a `//quiet unused function warning` above the  call would be nice,  but up to you if you think it's worth adding either